### ISSUE TITLE
Missing parameters

### ISSUE THREAD

**Issue by bkoyuncu** [ujjax/pred-lstm#2]
There are missing parameters in the given code I suppose,

1- class Encoder() , has missing arguments in __init__()
2- h = o * torch.tanh(F.conv2d(torch.cat((c,M), dim= ),self.weight_1x1)), dim is missing in ST-LSTM cell definition.
